### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 109)

### LINKED ISSUE #4914: [devtools-translate] Russian - What's New in DevTools (Chrome 109)
Post: https://developer.chrome.com/blog/new-in-devtools-109/
Translated by: Jan-21

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #4914

### CODE REVIEW

**Comment by jecfish** [comment]
Thanks Alena! @kateryna-prokopenko @webmaxru please take a look!

**Comment by jecfish** [comment]
Thanks Alena! @kateryna-prokopenko @webmaxru please take a look!

File: site/ru/blog/new-in-devtools-109/index.md [hunk 1]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
+Кроме того вы можете навигироваться между предложениями автодополнения при помощи <kbd>Ctrl</kbd> + <kbd>N</kbd> и <kbd>Ctrl + P</kbd> на MacOS, также как в [Emacs](https://www.gnu.org/software/emacs/). Например, вы можете напечатать `window.` в Консоли (Console) и затем использовать эти горячие клавиши для навигации.
 
 <!-- On top of that, DevTools now accepts <kbd>Arrow Right</kbd> for autocompletion only at the end of line. For example, an autocomplete dialog shows when you are editing something in the middle of the code. When you press the <kbd>Arrow Right</kbd> key, most likely, you want to set the cursor to the next position instead of autocomplete. This UX change better aligns with your authoring workflow. -->
+В добавок ко всему, DevTools теперь позволяет использовать клавишу <kbd>Стрелка вправо</kbd> для автодополнения только в конце строки. Например, вы редактируете что-то в середине строки кода. Когда вы нажимаете клавишу <kbd>Стрелка вправо</kbd> вы, скорее всего, ожидаете, что курсор переместится на следующий символ вместо вставки автодополнения. Это изменение UX лучше согласуется с вашим рабочим процессом редактирования.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:89]:
```suggestion
В добавок ко всему, DevTools теперь позволяет использовать клавишу <kbd>Стрелка вправо</kbd> для автодополнения только в конце строки. Например, вы редактируете что-то в середине строки кода. Когда вы нажимаете клавишу <kbd>Стрелка вправо</kbd>, вы, скорее всего, ожидаете, что курсор переместится на следующий символ вместо вставки автодополнения. Это изменение UX лучше согласуется с вашим рабочим процессом редактирования.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 2]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 
 <!-- ## Improved JavaScript debugging {: #debugging } -->
+## Улучшена отладка JavaScript {: #debugging }
 
 <!-- These are some JavaScript debugging improvements in this release: -->
+Вот некоторые улучшения процесса отладки JavaScript в этом обновлении:
 
 <!-- - `new.target` is a meta-property that lets you detect whether a function or constructor was called using the new operator. You can now log `new.target` in the **Console** to check its value during debugging. Previously, it would return errors when you entered `new.target`. -->
-   {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/hKOEn03BZN2IUmWJ1Hho.png", alt="Show before and after comparison of new.target evaluation debugging.", width="800", height="499" %}
+- `new.target` это мета-свойство, позволяющее определить была ли функция или конструктор вызваны с помощью оператора new.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:105]:
```suggestion
- `new.target` это мета-свойство, позволяющее определить, были ли функция или конструктор вызваны с помощью оператора new.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 3]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
 
 
 <!-- ## [Experimental] Enhanced UX in managing breakpoints -->
+## [Экспертимент] Улучшенный UX в интерфейсе управления точками останова
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:152]:
```suggestion
## [Эксперимент] Улучшенный UX в интерфейсе управления точками останова
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 4]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
 
 {% Aside %}
 <!-- To enable the experiment, check **Enable re-designed Breakpoint Sidebar Pane in the Sources Panel** under **Settings** > **Experiments**. -->
+Чтобы включить этот эксперимент поставьте галочку напротив **Enable re-designed Breakpoint Sidebar Pane in the Sources Panel** в **Настройки** (Settings) > **Эксперименты** (Experiments).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:156]:
```suggestion
Чтобы включить этот эксперимент, поставьте галочку напротив **Enable re-designed Breakpoint Sidebar Pane in the Sources Panel** в **Настройки** (Settings) > **Эксперименты** (Experiments).
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 5]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
 {% endAside %}
 
 <!-- The current **Breakpoints** pane provides little visual aid in overseeing all breakpoints. On top of that, frequently used actions are hidden behind the context menu. -->
+Текущая панель **Точки останова** (Breakpoints) мало помогает при просмотре всех точек останова. Кроме того, часто используемые действия спрятаны в контестном меню.
 
 <!-- This experimental UX redesign aims at bringing structure into the **Breakpoints** pane and allow developers to have quick access to commonly used features, in particular editing and removing breakpoints. -->
+Экспериментальный редизайн UX нужен чтобы добавить структуры в панель **Точки останова** (Breakpoints) и предоставляет разработчикам быстрый доступ к основым функциям, в частности, к добавлению и удалению точки останова.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:163]:
```suggestion
Экспериментальный редизайн UX нужен, чтобы добавить структуры в панель **Точки останова** (Breakpoints) и предоставляет разработчикам быстрый доступ к основным функциям, в частности, к добавлению и удалению точки останова.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 6]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
 
 <!-- These are some highlights: -->
+Основные моменты:
 
 <!-- - Both pause options are in the **Breakpoints** pane and labeled with text to make it more self-explanatory. -->
+- Обе опции паузы теперь на панели **Точки останова** (Breakpoints) и подписаны текстом, чтобы стать более понятными.
 <!-- - Breakpoints are grouped by file, ordered by line/column number, and collapsible.** -->
+- Точки останова сгруппированы по файлам, отсортированы по номерам строк и колонок, а также могут быть свёрнуты.
 <!-- - New options to remove and edit breakpoint when hovering over a breakpoint or file name in the **Breakpoint** pane. -->
+- Новые опции удаления и правки точки останова при наведении курсора на ней или на имя файла в панели **Точки останова** (Breakpoints).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:173]:
```suggestion
- Новые опции удаления и правки точки останова при наведении курсора на нее или на имя файла в панели **Точки останова** (Breakpoints).
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 7]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
-Chromium issues: [1346231](https://crbug.com/1346231), [1324904](https://crbug.com/1324904) 
+Задачи в трекере Chromium: [1346231](https://crbug.com/1346231), [1324904](https://crbug.com/1324904) 
 
 
 <!-- ## [Experimental] Automatic in-place pretty print -->
+## [Эксперимент] Автоматическое красивое форматирование внутри файла
 
 {% Aside %}
 <!-- To enable the experiment, check **Automatically pretty print in the Sources panel** under **Settings** > **Experiments**. -->
+Чтобы включить этот эксперимент поставьте галочку напротив **Automatically pretty print in the Sources panel** в **Настройки** (Settings) > **Эксперименты** (Experiments).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:189]:
```suggestion
Чтобы включить этот эксперимент, поставьте галочку напротив **Automatically pretty print in the Sources panel** в **Настройки** (Settings) > **Эксперименты** (Experiments).
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 8]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
 {% endAside %}
 
 <!-- The **Sources** panel now automatically pretty print minified source files in-place. You can click on the **pretty print button `{ }` to underdo it. -->
+Во вкладке **Источники** (Sources) минифицированные файлы теперь автоматичекси форматируются. Вы можете нажать на кнопку автоформатирования `{ }` чтобы увидеть файл без форматирования.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:193]:
```suggestion
Во вкладке **Источники** (Sources) минифицированные файлы теперь автоматически форматируются. Вы можете нажать на кнопку автоформатирования `{ }`, чтобы увидеть файл без форматирования.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 9]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
 
 <!-- Previously, the **Sources** panel shows minified content by default. Developers need to click on the pretty print button manually to format the content. On top of that, the pretty printed content is not displayed in the same file, but in another `::formatted` tab. -->
+Ранее на вкладке **Источники** (Sources) минифицированный контент по умолчанию отображался без форматирования. Разработчиком нужно было нажать на кнопку автоформатирования чтобы удобно читать контент. Кроме того, ранее форматированный контент отображался не в том же файле, а на новой вкладке `::formatted`.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:196]:
```suggestion
Ранее на вкладке **Источники** (Sources) минифицированный контент по умолчанию отображался без форматирования. Разработчикам нужно было нажать на кнопку автоформатирования, чтобы удобно читать контент. Кроме того, ранее форматированный контент отображался не в том же файле, а на новой вкладке `::formatted`.
```

File: site/ru/_partials/devtools/whats-new.md [hunk 10]
```diff
@@ -5,16 +5,15 @@
...
-* [[Experimental] Automatic in-place pretty print](/ru/blog/new-in-devtools-109/#pretty-print)
- #}
+### Chrome 109 {: #chrome109
+
+* [Recorder: опции копирования для шага, воспроизведение на странице, контекстное меню шага](/ru/blog/new-in-devtools-109/#recorder)
+* [Показ настоящих названий функций в записях производительности](/ru/blog/new-in-devtools-109/#performance)
+* [Новые горячие клавиши во вкладках Консоль (Console) и Источники (Sources)](/ru/blog/new-in-devtools-109/#keyboard-shortcuts)
+* [Улучшена отладка JavaScript](/ru/blog/new-in-devtools-109/#debugging)
+* [Другие важные моменты](/ru/blog/new-in-devtools-109/#misc)
+* [[Экспертимент] Улучшенный UX в интерфейсе управления точками останова](/ru/blog/new-in-devtools-109/#breakpoint-redesign)
```
Comment (webmaxru) [inline site/ru/_partials/devtools/whats-new.md:15]:
```suggestion
* [[Эксперимент] Улучшенный UX в интерфейсе управления точками останова](/ru/blog/new-in-devtools-109/#breakpoint-redesign)
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 11]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
+<!-- Не уверена, что «in-page replay» корректно перевести как «воспроизведение на странице» -->
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/uCqjrGj716ZbDJ4N37dl.png", alt="New copy options in the Recorder panel.", width="800", height="615" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/uCqjrGj716ZbDJ4N37dl.png", alt="Новые опции копирования во вкладке Recorder.", width="800", height="615" %}
 
 <!-- Open an existing user flow in the **Recorder**. Previously, when you replayed the user flow, DevTools would always start the replay by navigating to or reloading the page. -->
+Откройте существующий пользовательский сценарий во вкладке **Recorder**. Ранее, когда вы запускали сценарий, воспроизведение в DevTools всегда начиналось с перехода или перезагрузки страницы.
 
 <!-- With the latest updates, the **Recorder** shows the navigation step separately. You can right-click and remove it to perform in-page replay!  -->
+С последним обновлением **Recorder** теперь показывает шаг навигации отдельно. Вы можете нажать правой клавишей мыши и удалить его для воспроизведения на странице.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:38]:
```suggestion
С последним обновлением **Recorder** показывает шаг навигации отдельно. Вы можете нажать правой клавишей мыши и удалить его для воспроизведения на странице.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 12]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 
 <!-- Apart from that, you can right-click a step and copy it to the clipboard in the **Recorder* panel instead of exporting the whole user flow. It works with [extensions](https://goo.gle/recorder-extension) too. For example, try to copy a step as a [Nightwatch Test](https://bit.ly/nightwatch-recorder) script. With this feature, you can update any existing script with ease. -->
+Кроме того вы можете нажать правой клавишей мыши на шаг и скопировать его в буфер обмена во вкладке **Recorder** вместо того, чтобы экспортировать весь пользовательский сценарий. Это работает и с [расширениями](https://goo.gle/recorder-extension) тоже. Например, попробуйте скопировать шаг как скрипт для [Nightwatch Test](https://bit.ly/nightwatch-recorder). С помощью этой функции вы можете легко обновить любой существующий скрипт.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:41]:
```suggestion
Кроме того, вы можете нажать правой клавишей мыши на шаг и скопировать его в буфер обмена во вкладке **Recorder** вместо того, чтобы экспортировать весь пользовательский сценарий. Это работает и с [расширениями](https://goo.gle/recorder-extension) тоже. Например, попробуйте скопировать шаг как скрипт для [Nightwatch Test](https://bit.ly/nightwatch-recorder). С помощью этой функции вы можете легко обновить любой существующий скрипт.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 13]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 
-Chromium issues: [1322313](https://crbug.com/1322313), [1351649](https://crbug.com/1351649), [1322313](https://crbug.com/1322313), [1339767](https://crbug.com/1339767)
+Задачи в трекере Chromium: [1322313](https://crbug.com/1322313), [1351649](https://crbug.com/1351649), [1322313](https://crbug.com/1322313), [1339767](https://crbug.com/1339767)
 
 
 <!-- ## Show actual function names in performance’s recordings {: #performance } -->
+## Показ настоящих названий функций в записях производительности {: #performance }
 
 <!-- The **Performance** panel now shows the actual function names and their sources in the trace if there’s a sourcemap. -->
+Во вкладке **Производительность** (Performance) теперь показыаются фактические имена функций и их источник в трассировке, если есть карта источников.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:54]:
```suggestion
Во вкладке **Производительность** (Performance) теперь показываются фактические имена функций и их источник в трассировке, если есть карта источников.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 14]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/9pHMVM1ARXrlyLoTziVA.png", alt="Show before and after comparison of function names display in the Performance panel.", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/9pHMVM1ARXrlyLoTziVA.png", alt="Как имена функций показываются во вкладке Производительность: до и после обновления.", width="800", height="509" %}
 
 <!-- In this example, a source file is minified during production. For example, the `sayHi` function is minified as `n`, and the `takeABreak` function is minified as `o` in this [demo](https://clinquant-mousse-2f2396.netlify.app/). -->
+В этом примере исходный файл был минифицирован по время сборки. Например, имя функции `sayHi` было минифицировано до `n` и `takeABreak` было минифицировано до `o` в этом [демо](https://clinquant-mousse-2f2396.netlify.app/).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:59]:
```suggestion
В этом примере исходный файл был минифицирован по время сборки. Например, имя функции `sayHi` было минифицировано до `n`, и `takeABreak` было минифицировано до `o` в этом [демо](https://clinquant-mousse-2f2396.netlify.app/).
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 15]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ywER8cdQUNYrdAaBJTKT.png", alt="Show files before and after minfication.", width="800", height="392" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ywER8cdQUNYrdAaBJTKT.png", alt="Показ файла до и после минификации.", width="800", height="392" %}
 
 <!-- Previously, when you recorded a trace in the **Performance** panel, the trace only showed you the minified function names. This made it harder to debug.  -->
+Ранее, когда вы записывали трассировку во вкладке **Производительность** (Performance), в трассировке показывались только минифицированные имена функций. Это затрудняло отладку.
 
 <!-- With the latest changes, DevTools now reads the source map and shows the actual function names and source location.  -->
+С последним обновлением DevTools теперь читают карту источников и показывают фактические имена функций и их положение в исходниках.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-109/index.md:67]:
```suggestion
С последним обновлением DevTools читают карту источников и показывают фактические имена функций и их положение в исходниках.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 16]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/686acb9789020a511405a53a13ad754a7e928c99 #}
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/09c3ceaa1605b29d1074d0cf310958bdb823149d #}
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/6468c740419d01d4e13c9ad914001959e78ca782 #}
 
-Chromium issue: [1167965](https://crbug.com/1167965), [1172535](https://crbug.com/1172535),  [1371585](https://crbug.com/1371585). [1369503](https://crbug.com/1369503)
+Задачи в трекере Chromium: [1167965](https://crbug.com/1167965), [1172535](https://crbug.com/1172535),  [1371585](https://crbug.com/1371585). [1369503](https://crbug.com/1369503)
 
 
 <!-- ## Improved JavaScript debugging {: #debugging } -->
+## Улучшена отладка JavaScript {: #debugging }
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:99]:
```suggestion
## Улучшенная отладка JavaScript {: #debugging }
```
Please also update in the contents list above

File: site/ru/blog/new-in-devtools-109/index.md (hunk 2 above)
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:105]: Please translate the whole passage :)

File: site/ru/blog/new-in-devtools-109/index.md [hunk 17]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
+- `new.target` это мета-свойство, позволяющее определить была ли функция или конструктор вызваны с помощью оператора new.
+   {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/hKOEn03BZN2IUmWJ1Hho.png", alt="Показано сравнение до и после внедрения new.target", width="800", height="499" %}
 <!-- - A `WeakRef` object lets you hold a weak reference to another object, without preventing that object from getting garbage-collected. DevTools now shows an inline preview for the value and evaluates the weak reference directly in the console during debugging. Previously, you had to explicitly call “deref” on them to resolve it. -->
-   {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/M7DP4bI7pA07oY7M21wF.png", alt="Show before and after comparison of WeakRef evaluation during debugging.", width="800", height="453" %} 
+- Объект `WeakRef` позволяет хранить нестрогую ссылку на другой объект, не препятствуя работе сборщика мусора. DevTools теперь показывает предварительный просмотр значения и выполняет обработку слабой ссылки непосредственно в консоли во время отладки. Ранее для решения этой проблемы необходимо было явно вызывать для них команду "deref".
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:108]:
```suggestion
- Объект `WeakRef` позволяет хранить слабую ссылку на другой объект, не препятствуя работе сборщика мусора. DevTools теперь показывает предварительный просмотр значения и выполняет обработку слабой ссылки непосредственно в консоли во время отладки. Ранее для решения этой проблемы необходимо было непосредственно вызывать для них команду "deref".
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 18]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
+   {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/M7DP4bI7pA07oY7M21wF.png", alt="Показано сравнение до и после появления WeakRef.", width="800", height="453" %} 
 <!-- - Fixed inline preview for shadowed variable. Previously, the display value was incorrect.  -->
-   {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XHL8pnBxhZ65ni7zYV0Q.png", alt="Show before and after comparison inline preview for shadowed variable.", width="800", height="519" %} 
+- Исправлен предварительный просмотр изменяемой переменной. Ранее значение отображалось некорректно.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:111]:
```suggestion
- Исправлен встроенный предварительный просмотр изменяемой переменной. Ранее значение отображалось некорректно.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 19]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Другие важные моменты {: #misc }
 
 <!-- These are some noteworthy fixes in this release: -->
+Вот ещё несколько заслуживающих внимания исправлений в этом выпуске:
 
 <!-- - Support more hints for inactive CSS properties in the **Styles** pane - inline height and width, flex and grid properties. ([1373597](https://crbug.com/1373597), [1178508](https://crbug.com/1178508), [1178508](https://crbug.com/1178508),[1178508](https://crbug.com/1178508)) -->
+- Поддержка дополнительных подсказок для неактивнымх свойств CSS на панели **Стили** (Styles) — высота и ширина у строчных элементов, свойства флексбоксов и гридов. ([1373597](https://crbug.com/1373597), [1178508](https://crbug.com/1178508), [1178508](https://crbug.com/1178508),[1178508](https://crbug.com/1178508))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:132]:
```suggestion
- Поддержка дополнительных подсказок для неактивных свойств CSS на панели **Стили** (Styles) — высота и ширина у строчных элементов, свойства флексбоксов и гридов. ([1373597](https://crbug.com/1373597), [1178508](https://crbug.com/1178508), [1178508](https://crbug.com/1178508),[1178508](https://crbug.com/1178508))
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 20]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 <!-- - Fixed syntax highlighting. It was not working properly since the recent [code editor](https://codemirror.net/) upgrade in DevTools. ([1290182](https://crbug.com//1290182)) -->
+- Исправлена подсветка синтаксиса. Он работал неверно с момента обновления [редактора кода](https://codemirror.net/) в DevTools. ([1290182](https://crbug.com//1290182))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:134]:
```suggestion
- Исправлена подсветка синтаксиса. Она работал неверно с момента обновления [редактора кода](https://codemirror.net/) в DevTools. ([1290182](https://crbug.com//1290182))
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 21]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 <!-- - Capture input change events properly after on blur event in the **Recorder**. ([1378488](https://crbug.com/1378488)) -->
+- Правильное отслеживание событий изменения в поле ввода после события blur в **Recorder**. ([1378488](https://crbug.com/1378488))
 <!-- - Update Puppeteer replay script on export for better debugging experience in the **Recorder**. ([1351649](https://crbug.com/1351649)) -->
+- Обновлён экспорт скрипта для Puppeteer для улучшения опыта отладки в **Recorder**. ([1351649](https://crbug.com/1351649))
 <!-- - Support record and replay in the **Recorder** for remote debugging. ([1185727](https://crbug.com/1185727))  -->
+- Поддержка записи и ответа в **Recorder** для удалённой отладки. ([1185727](https://crbug.com/1185727))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:140]:
```suggestion
- Поддержка записи и воспроизведения в **Recorder** для удалённой отладки. ([1185727](https://crbug.com/1185727))
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 22]
```diff
@@ -4,111 +4,142 @@ title: "Новинки DevTools (Chrome 109)"
...
 <!-- - Fixed parsing of special CSS variable names in `var()`. Previously, DevTools didn't support parsing variables with escaped characters like `var(--fo\ o)`. , ([1378992](https://crbug.com/1378992)) -->
+- Исправлен парсинг имён специальных CSS-переменных внутри `var()`. Ранее DevTools не поддерживали парсинг переменных с определёнными символами, например, `var(--fo\ o)`. , ([1378992](https://crbug.com/1378992))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:142]:
```suggestion
- Исправлен парсинг имён специальных CSS-переменных внутри `var()`. Ранее DevTools не поддерживал парсинг переменных с экранированными символами, например, `var(--fo\ o)`. , ([1378992](https://crbug.com/1378992))
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 23]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
 <!-- ## [Experimental] Enhanced UX in managing breakpoints -->
+## [Экспертимент] Улучшенный UX в интерфейсе управления точками останова
 
 {% Aside %}
 <!-- To enable the experiment, check **Enable re-designed Breakpoint Sidebar Pane in the Sources Panel** under **Settings** > **Experiments**. -->
+Чтобы включить этот эксперимент поставьте галочку напротив **Enable re-designed Breakpoint Sidebar Pane in the Sources Panel** в **Настройки** (Settings) > **Эксперименты** (Experiments).
 {% endAside %}
 
 <!-- The current **Breakpoints** pane provides little visual aid in overseeing all breakpoints. On top of that, frequently used actions are hidden behind the context menu. -->
+Текущая панель **Точки останова** (Breakpoints) мало помогает при просмотре всех точек останова. Кроме того, часто используемые действия спрятаны в контестном меню.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:160]:
```suggestion
Нынешняя панель **Точки останова** (Breakpoints) мало помогает при просмотре всех точек останова. Кроме того, часто используемые действия спрятаны в контекстном меню.
```

File: site/ru/blog/new-in-devtools-109/index.md (hunk 5 above)
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:163]:
```suggestion
Экспериментальный редизайн UX привносит структуру в панель **Точки останова** (Breakpoints) и предоставляет разработчикам быстрый доступ к основым функциям, в частности, к добавлению и удалению точки останова.
```

**Comment by jecfish** [comment]
Thanks Maxim. Over to you, Alena!

File: site/ru/blog/new-in-devtools-109/index.md [hunk 24]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
 
 <!-- This experimental UX redesign aims at bringing structure into the **Breakpoints** pane and allow developers to have quick access to commonly used features, in particular editing and removing breakpoints. -->
+Экспериментальный редизайн UX нужен чтобы добавить структуры в панель **Точки останова** (Breakpoints) и предоставляет разработчикам быстрый доступ к основым функциям, в частности, к добавлению и удалению точки останова.
 
 <!-- These are some highlights: -->
+Основные моменты:
 
 <!-- - Both pause options are in the **Breakpoints** pane and labeled with text to make it more self-explanatory. -->
+- Обе опции паузы теперь на панели **Точки останова** (Breakpoints) и подписаны текстом, чтобы стать более понятными.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:169]:
```suggestion
- Обе опции паузы теперь на панели **Точки останова** (Breakpoints). Они имеют текстовые метки, которые делают их интуитивно понянятными.
```

File: site/ru/blog/new-in-devtools-109/index.md [hunk 25]
```diff
@@ -118,45 +149,58 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
 <!-- - Breakpoints are grouped by file, ordered by line/column number, and collapsible.** -->
+- Точки останова сгруппированы по файлам, отсортированы по номерам строк и колонок, а также могут быть свёрнуты.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:171]:
```suggestion
- Точки останова сгруппированы по файлам, отсортированы по номерам строк или колонок и могут быть свернуты и развернуты.
```

File: site/ru/blog/new-in-devtools-109/index.md (hunk 8 above)
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-109/index.md:193]:
```suggestion
Во вкладке **Источники** (Sources) минифицированные файлы теперь автоматичекси форматируются. Вы можете нажать на кнопку автоформатирования `{ }` чтобы отменить это.
```

**Review by kateryna-prokopenko — COMMENTED** [review]
Thanks Alena! Please also apply Maxim's suggestions.

**Comment by jecfish** [comment]
Thanks Kateryna, over to @solarrust!

File: site/ru/blog/new-in-devtools-109/index.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (solarrust) [reply site/ru/blog/new-in-devtools-109/index.md:67]: But it has ’s’ in the end 🤔 So, for me, it is plural

**Review by jecfish — APPROVED** [review]

**Pull request merged**